data/core/help.cfg

```diff
@@ -93,7 +93,7 @@
     [topic]
         id=..introduction
         title= _ "Introduction"
-        text="<img>src=misc/logo-bg.png~BLIT(misc/logo.png) align=middle</img>" + _ "<br/><italic>text='Battle for Wesnoth'</italic> is a turn-based fantasy strategy game somewhat unusual among modern strategy games. While other games strive for complexity, <italic>text='Battle for Wesnoth'</italic> strives for simplicity of both rules and gameplay. This does not make the game simple, however — from these simple rules arises a wealth of strategy, making the game easy to learn but a challenge to master.
+        text="<img>src=misc/logo-bg.png~BLIT(misc/logo.png) align=middle</img><br/>" + _ "<italic>text='Battle for Wesnoth'</italic> is a turn-based fantasy strategy game somewhat unusual among modern strategy games. While other games strive for complexity, <italic>text='Battle for Wesnoth'</italic> strives for simplicity of both rules and gameplay. This does not make the game simple, however — from these simple rules arises a wealth of strategy, making the game easy to learn but a challenge to master.
```

Comment:
Since you're already working on this, would be good if you could change this to the new markup syntax, saves some work later.
```suggestion
        text="<img src=misc/logo-bg.png~BLIT(misc/logo.png) align=middle /><br/>" + _ "<italic>Battle for Wesnoth</italic> is a turn-based fantasy strategy game somewhat unusual among modern strategy games. While other games strive for complexity, <italic>Battle for Wesnoth</italic> strives for simplicity of both rules and gameplay. This does not make the game simple, however — from these simple rules arises a wealth of strategy, making the game easy to learn but a challenge to master.
```
Ref: [Help Markup](https://wiki.wesnoth.org/Help_markup)

Reply:
Feel free to update markup to the new style anywhere else as well, if you feel like it.

Reply:
That's probably best left as a separate task.